src/actions/plugins/easymotion/easymotion.ts

```diff
@@ -243,22 +232,18 @@ export class EasyMotion {
         after: {
           contentText: keystroke,
           backgroundColor: this.getEasymotionMarkerBackgroundColor(),
-          height: `${configuration.easymotionMarkerHeight}px`,
```

Comment:
If someone specifies a height now it is ignored. I am not "opposed" to this but it makes it a bit tricky to clean up the configuration for these markers, I don't think too many people have set them, but for a few people this could be a regression if they like the way they setup their markers.

Reply:
Okay, I'll get it back then.

Reply:
Yes ! Love this ❤️